### PR TITLE
Add dev command and add project info into package.json

### PR DESCRIPTION
**Pull request by fxnoob**
### Description
to build the extension, we need to run `./bin/build.sh` which takes longer to make a build 
### Description
to build the extension we run `npm run extension` which creates build with less overhead. 
### Screenshots
Screenshot 2019-08-26 at 9 30 31 AM

### CODE REVIEW

**Comment by fxnoob** [comment]
closes #6

**Comment by naturetime10** [comment]
https://github.com/byliuyang/short-ext/issues/6#issuecomment-524716234

File: tsconfig.json [hunk 1]
```diff
@@ -13,7 +13,7 @@
     // "declarationMap": true,                /* Generates a sourcemap for each corresponding '.d.ts' file. */
     // "sourceMap": true,                     /* Generates corresponding '.map' file. */
     // "outFile": "./",                       /* Concatenate and emit output to single file. */
-    "outDir": "./build",
```
Comment (byliuyang) [inline tsconfig.json]: Please keep build directory as we discussed.

File: package.json [hunk 2]
```diff
@@ -1,4 +1,16 @@
...
+  "version": "0.0.4",
+  "description": "Visit short links with less typing",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/byliuyang/short-ext.git"
+  },
+  "scripts": {
+    "dev": "yarn run cp && tsc --watch",
+    "build": "yarn run cp && tsc",
+    "cp": "cp -r icons extension/ && cp manifest.json extension/"
```
Comment (naturetime10) [inline package.json]:
```suggestion
    "cp:assets": "cp -r icons manifest.json build/"
```

File: package.json [hunk 3]
```diff
@@ -1,4 +1,16 @@
 {
+  "name": "Short",
+  "version": "0.0.4",
+  "description": "Visit short links with less typing",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/byliuyang/short-ext.git"
+  },
+  "scripts": {
+    "dev": "yarn run cp && tsc --watch",
```
Comment (naturetime10) [inline package.json]:
```suggestion
    "dev": "yarn build --watch",
```

File: bin/build.sh [hunk 4]
```diff
@@ -9,4 +9,4 @@ cp manifest.json build/short-ext/
 cp -r icons build/short-ext/
 
 cd build
-zip -r short-ext.zip short-ext/* -x "*.DS_Store"
```
Comment (byliuyang) [inline bin/build.sh]: yarn build

File: README.md [hunk 5]
```diff
@@ -12,8 +12,9 @@ Install from [Chrome Web Store](https://s.time4hacks.com/r/ext)
 
 ## Getting Started
 ### Build
-```bash
-./bin/build.sh
+```
+yarn
+yarn run build
 ```
```
Comment (naturetime10) [inline README.md:19]: Please add development instructions here

File: README.md [hunk 6]
```diff
@@ -12,8 +12,9 @@ Install from [Chrome Web Store](https://s.time4hacks.com/r/ext)
 
 ## Getting Started
 ### Build
-```bash
-./bin/build.sh
+```
+yarn
```
Comment (naturetime10) [inline README.md:16]:
```suggestion
./bin/build
```

File: README.md [hunk 7]
```diff
@@ -12,8 +12,9 @@ Install from [Chrome Web Store](https://s.time4hacks.com/r/ext)
...
+yarn run build
```
Comment (naturetime10) [inline README.md:17]:
```suggestion

```

File: tsconfig.json [hunk 8]
```diff
@@ -13,7 +13,7 @@
...
+    "outDir": "./extension",
```
Comment (naturetime10) [inline tsconfig.json]:
```suggestion
    "outDir": "./build",
```

File: package.json [hunk 9]
```diff
@@ -1,4 +1,16 @@
...
+    "build": "yarn run cp && tsc",
```
Comment (naturetime10) [inline package.json]:
```suggestion
    "build": "rm -rf build && yarn cp:assets && tsc",
```

**Review by naturetime10 — CHANGES_REQUESTED** [review]

File: package.json (hunk 2 above)
Comment (fxnoob) [reply package.json]: `cp -r icons manifest.json build/` throws error

Comment (naturetime10) [reply package.json]: @fxnoob It works for me on macOS. Did you create ```build/``` directory first?

Comment (naturetime10) [reply package.json]: Please try this: 
```json
"scripts": {
    "build": "yarn prep && yarn cp:assets && tsc",
    "prep": "rm -rf build && mkdir build",
    "dev": "yarn build --watch",
    "cp:assets": "cp -r icons manifest.json build/"
  },
```

File: bin/build.sh (hunk 4 above)
Comment (naturetime10) [inline bin/build.sh:12]: Just verified the following works:

```bash
#!/usr/bin/env bash

yarn build

mkdir -p build/short-ext

mv build/background.js build/short-ext/
cp manifest.json build/short-ext/
cp -r icons build/short-ext/

zip -r build/short-ext.zip build/short-ext/* -x "*.DS_Store"
```

File: README.md (hunk 7 above)
Comment (naturetime10) [inline README.md:17]:
```suggestion
yarn build
```

**Review by naturetime10 — APPROVED** [review]

**Pull request merged**